integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/testcases/standalone/awsLambda/AwsLambdaSupportTestCase.java

```diff
@@ -0,0 +1,69 @@
+package org.wso2.choreo.connect.tests.testcases.standalone.awsLambda;
+
+import io.netty.handler.codec.http.HttpHeaderNames;
+import netscape.javascript.JSObject;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.choreo.connect.tests.util.*;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+public class AwsLambdaSupportTestCase {
+
+    protected String jwtTokenProd;
+
+    @BeforeClass(description = "initialise the setup")
+    void start() throws Exception {
+        jwtTokenProd = TokenUtil.getJwtForPetstore(TestConstant.KEY_TYPE_PRODUCTION,
+                null, false);
+    }
+
+    @Test(description = "Test invoke Aws Lambda API 1")
+    public void testInvokeAwsLambda1() throws Exception {
+
+        Map<String, String> headers = new HashMap<>();
+        //test endpoint with token
+        headers.put(HttpHeaderNames.AUTHORIZATION.toString(), "Bearer " + jwtTokenProd);
```

Comment:
Let's move make the map as a instance variable and after that we can do the put operation inside the start() method.  Please consider this with the second test case as well.

Reply:
Fixed by e0f92a0